### ISSUE TITLE
Save location at boot.

### ISSUE THREAD

**Issue by Zilverfire** [fgsfdsfgs/openmw#10]
The read and write save data location for the save game at boot are \switch\openmw\data\global\saves, as nsp forwarders don't ask for the user profile, but if you open another game first or after you play Morrowind for a bit player 1 is given a user profile so the game will try to read and write save data from F:\switch\openmw\data\(user)\saves.
Not truly a bug, closer to an oversight but the game runs good, great work.

**Comment by terabyte25** [comment]
Maybe try and run the profile applet programmatically? Libnx does this somewhere in their web.c code, but I am not sure where.

**Comment by fgsfdsfgs** [comment]
Yes, I have noticed this. I don't think you can run the profile selection thing programmatically, I think it's part of qlaunch? However I am pretty sure that you can make a forwarder that lets you select a profile, because I've had a build of NSP hbmenu that did that.

**Comment by terabyte25** [comment]
@fgsfdsfgs As I said, libnx does it programmatically. No idea how. 

It is documented though: https://switchbrew.org/wiki/Profile_Selector

**Comment by terabyte25** [comment]
And it's in the code under "applet_playerselect" here:
https://github.com/switchbrew/libnx/blob/5a3eb542b23beb2f6f73ca5ce266dde6b2ad90a5/nx/include/switch/services/applet.h#L71 

There's a function that I found in the documentation that can load applets with the given ID (from the enum in the link above). But I'm not entirely sure how to use it.

**Comment by terabyte25** [comment]
Pull Request #12
